nemo/export/tensorrt_llm.py

```diff
@@ -167,6 +167,8 @@ def export(
         multiple_profiles: bool = False,
         gpt_attention_plugin: str = "auto",
         gemm_plugin: str = "auto",
+        fp8_quantized: bool = False,
```

Comment:
Is there way to detect if a ckpt is a fp8 ckpt by reading something from config?

Reply:
Yes, we can read if the checkpoint is in fp8 from `nemo_config`.

The idea behind the flags is:
- preserving the old behaviour,
- enabling deployment of fp8 checkpoints on non-fp8 hardware,
- in some [cases](https://github.com/NVIDIA/NeMo/blob/753c70e5c5cfc6acb7ecfb416374aed59d5d233c/nemo/export/quantize/quantizer.py#L120-L125) using FP8 KV-cache leads to accuracy regression,
- from the developmental point of view, it will be easier to isolate problems when some issues arise.

I've changed the default behaviour to auto-detection, that can be overridden by the flags.